.eslintrc

```diff
@@ -29,7 +29,7 @@
     # Rules below here should be enabled once violations are fixed.
     "react/forbid-prop-types": 0,
     "react/no-string-refs": 0,
-    "react/jsx-equals-spacing": 0,
+    "react/jsx-equals-spacing": [2, "never"],
```

Comment:
This is a rule that is enabled by default, so rather than changing the configuration, the entry for it should simply be removed so that we rely on the defaults. The goal here is to get as close to a default configuration as possible, rather than explicitly configure the disabled rules.

Reply:
Okay, noted.